Makefile

```
@@ -1,5 +1,5 @@
 # Gomander Build Makefile
-.PHONY: all clean windows darwin linux dmg docker-build dev help lint
+.PHONY: all clean windows darwin linux dmg docker-build docker-build-amd64 docker-build-arm64 deb deb-amd64 deb-arm64 dev help lint
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)